rbac/default-role-manager/role_manager.go

```diff
@@ -454,6 +455,11 @@ func NewDomainManager(maxHierarchyLevel int) *DomainManager {
 	return dm
 }
 
+func (dm *DomainManager) SetDomainHierarchyManager(dhm *RoleManager) {
```

Comment:
I am not sure if this is possible through the code, but I think this could have a default value like `e.rmMap['g2']`, so that users can define it, but do not necessarily need to.

Reply:
Unless a ``[domain_definition]`` is defined in the model like ``[role_definition]``, but redundant coding will greatly increase